packages/api-server/src/cliHandlers.ts

```diff
@@ -43,60 +43,55 @@ export const apiCliOptions = {
 export const apiServerHandler = async (options: ApiServerArgs) => {
   const { port, socket, apiRootPath } = options
   const tsApiServer = Date.now()
-  process.stdout.write(c.dim(c.italic('Starting API Server...\n')))
+  process.stdout.write(c.dim.italic('Starting API Server...\n'))
```

Comment:
Some places it was `c.dim.italic`; others `c.italic.dim`. I made them all `c.dim.italic`, and you can chain these, so anywhere I saw `c.dim(c.italic(...` it's now `c.dim.italic(...`, so, a little easier to read

Reply:
I wouldn't mind you switching to `console.log` tbh

Reply:
Done in https://github.com/redwoodjs/redwood/pull/9894/commits/0695ed84e37edfd2b9a63126ef728a6b631f04f3